### PR TITLE
sub second logging

### PR DESCRIPTION
**Pull request by acolombi**
This address issue #43

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/dropwizard-web-logger](https://github.com/palantir/dropwizard-web-logger), @acolombi! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: src/main/java/com/palantir/weblogger/WebLoggerResource.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ private JSONObject addEventName(JSONObject jsonEvent, String eventName) {
     }
 
     private JSONObject addTimestamp(JSONObject jsonEvent) {
-        SimpleDateFormat ft = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
+        SimpleDateFormat ft = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```
Comment (acolombi) [inline src/main/java/com/palantir/weblogger/WebLoggerResource.java:67]: adds sub-second level logging.

File: src/test/resources/server.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ server:
     - type: http
       port: 8000
 
-webLoggerEnabled: false
+webLoggerEnabled: true
```
Comment (acolombi) [inline src/test/resources/server.yml:8]: enables weblogger so the instructions in readme.md for com.palantir.weblogger.examples.Example work

**Comment by zoubeiri** [comment]
👍

**Comment by zoubeiri** [comment]
merge when ready

**Comment by acolombi** [comment]
I don't think I can merge it, right?

**Comment by zoubeiri** [comment]
hmm looks like circle is hung on something

**Comment by zoubeiri** [comment]
I'll merge as soon as circle is done

File: src/main/java/com/palantir/weblogger/WebLoggerResource.java (hunk 1 above)
Comment (sixinli) [reply src/main/java/com/palantir/weblogger/WebLoggerResource.java:67]: should be "sss"? (to match the casing for seconds?)

**Comment by sixinli** [comment]
can't find the build on circle, probably can just retrigger by changing some more thing in the PR

File: src/main/java/com/palantir/weblogger/WebLoggerResource.java (hunk 1 above)
Comment (sixinli) [reply src/main/java/com/palantir/weblogger/WebLoggerResource.java:67]: nvm casing matters :(

**Comment by qinfchen** [comment]
close this in favor of https://github.com/palantir/dropwizard-web-logger/pull/45

**Comment by qinfchen** [comment]
Circle project was configured to not build on fork and it is fixed now

**Pull request closed without merging**